src/appleseed/renderer/modeling/camera/orthographiccamera.cpp

```diff
@@ -251,14 +251,18 @@ namespace
                 "  film height                   %f\n"
                 "  near z                        %f\n"
                 "  shutter open                  %f\n"
-                "  shutter close                 %f",
+                "  shutter close                 %f\n"
```

Comment:
Nice! But I think it will be better if `close` and `open_end` order changed, same with input form.

Reply:
Really thanks for your patience!
I see it and should I also change `shutter open`  and `shutter close` to  `shutter open start` and `shutter close start` respectively?

Reply:
It doesn't seem necessary for me, but I'm not sure.